conf/locale/he/LC_MESSAGES/django.po

```diff
@@ -17741,6 +17741,14 @@ msgstr "חשב ציון מחדש רק אם הציון משתפר"
 msgid "Show the status for the tasks that you submitted for this problem."
 msgstr "הצג את סטטוס המשימות שהגשת לגבי בעיה זו."
 
+#: lms/templates/learner_dashboard/_dashboard_incomplete_profile_notification.html
+msgid "Your profile has missing fields!"
+msgstr ""
```

Comment:
@diegomillan Hebrew translations for these strings are available in the Google sheet attached to the ticket.  Could you please add them here?